<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="config_file" default="$(find maplab_features)/config/super_lidar.yaml"/>

  <node name="super" pkg="maplab_features" type="ros_interface.py" output="screen">
    <rosparam command="load" file="$(arg config_file)"/>
    <param name="pca_pickle_path" value="$(find maplab_features)/config/superpoint_pca.pkl"/>
  </node>

  <node name="lidar_image" pkg="maplab_features" type="lidar_image_interface.py" output="screen">
    <rosparam command="load" file="$(arg config_file)"/>
    <param name="lidar_calibration" value="$(find maplab_features)/config/calibrations/hilti-lidar.csv"/>
  </node>
</launch>
